client/src/pages/AddReview/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import propTypes from 'prop-types';
+// https://www.npmjs.com/package/react-typeform-embed
+import { ReactTypeformEmbed } from 'react-typeform-embed';
+
+import { Button } from '../../components';
+import urls from './urls';
+
+import './style.css';
+
+let typeformEmbed = null;
+const openForm = () => typeformEmbed.typeform.open();
+
+const AddReview = ({ type, sitspotId }) => (
+  <>
+    <div className="header-img header-container" />
```

Comment:
where does `header-container` come from? Don't see it in the component's css file (and why have two instead of one?). Also, `header-img` and `header-container` are pretty generic names, if those classes are generic (used in multiple places across the project), they should be defined in the top-level css file, if they are however specific for this component only, pls use a more specific name (`add-review-header`...)

Reply:
this comes from the index.css, bc we use it in almost every page component